### DISCUSSION TITLE
Discord Sever

### DISCUSSION THREAD

**Discussion by N8python** [enable3d/enable3d#124] [Ideas]
While Github Discussion is great for questions and help, having an enable3d discord would be helpful for live chat and discussion.
What do you all think?

**Comment by vitalspace** [comment]
I had already proposed it but I don't know if Yandeu wants to do it.

**Reply by N8python** [reply]
I see. Hopefully Yandeu leaves a comment.

**Comment by yandeu** [comment]
It would help, yes. But I don't use discord. And I don't like chatting. You can create a _unofficial_ channel if you want. And I could place a link somewhere. But I will probably not join. I prefere to stay offline 😎

**Reply by N8python** [reply]
I see. You prefer a more formal back-and-forth on questions - totally understandable. I think once the community gets a bit bigger I might create an unofficial discord server, if that's okay with you.

**Reply by yandeu** [reply]
Ok 👍
